packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -266,6 +268,15 @@ try {
 		});
 	}
 
+	// Enable the WordPress debug mode
+	await defineWpConfigConsts(php, {
+		consts: {
+			WP_DEBUG: true,
+			WP_DEBUG_LOG: true,
```

Comment:
@adamziel I'm not sure if this should be set to true by default. true returns all logs including warnings and notices, so it's noisy, but that's the point of debugging.

Reply:
Hm 🤔 Let's start with this set to `true`, and scale back if it produces too much output.